libvirt/resource_libvirt_network_test.go

```diff
@@ -604,14 +468,50 @@ func TestAccLibvirtNetwork_Autostart(t *testing.T) {
 					autostart = false
 				}`, randomNetworkResource, randomNetworkName),
 				Check: resource.ComposeTestCheckFunc(
-					networkExists("libvirt_network."+randomNetworkResource, &network),
+					testAccCheckNetworkExists("libvirt_network."+randomNetworkResource, &network),
 					resource.TestCheckResourceAttr("libvirt_network."+randomNetworkResource, "autostart", "false"),
 				),
 			},
 		},
 	})
 }
 
+// //////////////////////////////////////////////////////////////////
```

Comment:
i have created this file in past for `support` func  https://github.com/dmacvicar/terraform-provider-libvirt/blob/master/libvirt/acceptance_tests_functions_helpers.go

Reply:
we could move everything there so we avoid duplicata

Reply:
I think that file will not be linked with all the `*_test.go` files as it doesn't end with `_test` (at least my editor is complaining about that).

Reply:
But there is a `provider_test.go` where these things could go. I will move these functions there...

Btw, @MalloZup, next time please choose a name shorter than `acceptance_tests_functions_helpers.go` :joy:

Reply:
i was in the phase where i was thinking more is better. Now i am more in the middle :smile: